examples/greeter-extension/src/__tests__/greeter-extension.acceptance.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/example-greeter-extension
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Application, bind, createBindingFromClass} from '@loopback/core';
+import {expect} from '@loopback/testlab';
+import chalk from 'chalk';
+import {
+  asGreeter,
+  Greeter,
+  GreeterComponent,
+  GreetingService,
+  GREETING_SERVICE,
+} from '..';
+
+describe('greeter-extension-pont', () => {
+  let app: Application;
+  let greetingService: GreetingService;
+
+  beforeEach(givenAppWithGreeterComponent);
+  beforeEach(findGreetingService);
+
+  it('greets by language', async () => {
+    let msg = await greetingService.greet('en', 'Raymond');
+    expect(msg).to.eql('Hello, Raymond');
+    msg = await greetingService.greet('zh', 'Raymond');
+    expect(msg).to.eql('Raymond，你好！');
+  });
+
+  it('supports options for the extension point', async () => {
+    // Configure the extension point
+    app.bind('services.GreetingService.options').to({color: 'blue'});
+    const _greetingService = await app.get(GREETING_SERVICE);
```

Comment:
Why a name starting with an underscore?

```suggestion
    greetingService = await app.get(GREETING_SERVICE);
```